src/redux.js

```diff
@@ -721,24 +721,33 @@ export function getUniqueOptionsByColumn(state) {
   return uniqueOptionsByColumn;
 }
 
-export function getRangesByColumn(state) {
-  let rangesByColumn = {};
+export function getExtremumsByColumn(state) {
+  let extremumsByColumn = {};
   getNumericalColumns(state).map(
-    column => (rangesByColumn[column] = getRange(state, column))
+    column => (extremumsByColumn[column] = getExtremums(state, column))
   );
-  return rangesByColumn;
+  return extremumsByColumn;
 }
 
-export function getRange(state, column, shouldReturnRange = true) {
-  let range = {};
-  range.max = Math.max(...state.data.map(row => parseFloat(row[column])));
-  range.min = Math.min(...state.data.map(row => parseFloat(row[column])));
+function getMaximumValue(state, column) {
+  return Math.max(...state.data.map(row => parseFloat(row[column])));
+}
 
-  if (shouldReturnRange) {
-    range.range = range.max - range.min;
-  }
+function getMinimumValue(state, column) {
+  return Math.min(...state.data.map(row => parseFloat(row[column])));
+}
+
+function getRange(maximumValue, minimumValue) {
+  return Math.abs(maximumValue - minimumValue);
+}
+
+export function getExtremums(state, column) {
```

Comment:
could you add a block comment explaining what the `extremums` return value means?

imo it would be worthwhile to actually define an `extremumsType` `PropType` so you can validate `Predict.propTypes.extremumsByColumn`